createPrsToGenerateReleaseDrafterWorkflow.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+# Release Drafter Workflow Creator CLI
+# ===================================
+# This script creates pull requests to add a Release Drafter workflow to each repository
+# in the axonivy-market GitHub Organization.
+# Using https://cli.github.com/
+
+org="axonivy-market"
+
+ignored_repos=(
+  "market-up2date-keeper"
+  "market.axonivy.com"
+  "market-monitor"
+  "market"
+  "demo-projects"
```

Comment:
Portal team also working on this stuff? do we need to check with them to ignore portal also? then portal will handle it?

Reply:
After checking with Portal, I see they agree using workflow from the github-workflows